src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
 import React from 'react';
+import { Comment } from '../../react-app-env';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return (
+    <>
+      {(comments.length)
+        ? (
+          comments.map(comment => (
+            <div className="CommentList">
+              <CommentInfo comment={comment} />
+            </div>
+          ))
```

Comment:
also semantics

Reply:
I don't think semantics are mandatory in this task

Reply:
\+